onnxruntime/contrib_ops/cpu/transformers/beam_search_shared.h

```diff
@@ -43,6 +44,18 @@ struct IBeamSearchCpuState {
   gsl::span<float> final_beam_scores;  // shape (batch_size, num_beams)
 };
 
+template <typename T>
+struct IGreedySearchState {
```

Comment:
shall we change the filename from beam_search_shared.h to generation_shared.h, or move IGreedySearchState to another file like greedy_search_shared.h?

Reply:
will change the name to generation_shared.h